docs/source/tutorials/axiparabola.ipynb

```diff
@@ -0,0 +1,241 @@
+{
```

Comment:
It is easier to review this file by looking at the rendering here:
https://lasydoc--254.org.readthedocs.build/en/254/tutorials/axiparabola.html